test/test_derivatives.jl

```diff
@@ -16,6 +16,9 @@ else
     end
 end
 
+const datadir = joinpath(Pkg.dir("Celeste"), "test", "data")
```

Comment:
I might be redefining a constant here, but it's nice to run just one file or tests without having to run all the tests. (Is there a better way?)